### ISSUE TITLE
[FEAT]: Expert Train & Inference for Sarc dataset with ALBEF

### ISSUE THREAD

**Issue by Jasonqi146** [lwaekfjlk/mmodel-soup#87]
### Description

Use Sarc training split of R, U, and S to train model ALBEF seperately and save to
1.  Sarc R finetuned ALBEF
2.  Sarc U finetuned ALBEF
3.  Sarc S finetuned ALBEF

Then record the performance of the three models on the test set.
